docs/basics/_examples/DL-101-135-103

```diff
@@ -62,7 +62,7 @@ NOTE
   -D DESCRIPTION, --description DESCRIPTION
                         short description to use for a dataset location. Its
                         primary purpose is to help humans to identify a
-                        dataset copy (e.g., "mike's dataset on lab server").
+                        dataset copy (e.g. "mike's dataset on lab server").
```

Comment:
Note that this is an automatically generated file. Every file in ``_examples`` contains the contents of a command specified within the ``runrecord`` directive, in this case its

```
.. runrecord:: _examples/DL-101-135-103
   :language: console
   :workdir: dl-101/DataLad-101
   :lines: 1-40

   $ datalad get --help
```

With every ``make clean-examples`` (often necessary when writing new chapters to check if the full things still builds) and subsequent ``make``, the comma will be put back as it is part of the ``datalad get --help`` description. We can delete the comma in here, but need to be mindful of not accidentally saving the file whenever building the book again from the very start (i.e. after all examples are deleted).
If you feel really strongly about the comma after e.g. we could also change all other e.g. to include a comma - I just found out that "e.g." is British English, and "e.g.," American, and I was trying anyway to use American English consistently.

Reply:
Oh sorry, missed that. Will remove this change

Then iirc i.e. also should follow, shouldn't it?